deploy.sh

```diff
@@ -51,78 +36,28 @@ function get_shorthash {
   echo "${shorthash}"
 }
 
-function preprocess_and_publish {
+function preprocess_and_deploy {
 
   yarn lint || abort "Lint failure"
   yarn build || abort "Failed to build"
 
   local shorthash
   shorthash=$(get_shorthash)
 
-  local remote_url
-  remote_url=$(git remote get-url "${GH_REMOTE}")
+  local commitMessage="Update using ${SOURCE_BRANCH}/${shorthash}"
 
-  cd ${WEBSITE_DIR_PATH} || abort "Failed to navigate to website directory"
+  node "$GH_PAGES_BIN_PATH" --dist "$WEBSITE_DIR_PATH" --message "$commitMessage" || abort "gh-pages failed to deploy"
```

Comment:
Might it be simpler here to use `yarn gh-pages`?

```suggestion
  yarn gh-pages --dist "$WEBSITE_DIR_PATH" --message "$commitMessage" || abort "gh-pages failed to deploy"
```

Reply:
Ah! I forgot this was a possibility. Pushing this up.